src/components/NavBar/NavBar.css

```diff
@@ -0,0 +1,36 @@
+.nav-container {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  width: 100%;
+  font-family: 'Quicksand', sans-serif;
+}
+
+.nav-icon-and-h1 {
```

Comment:
We can make these class names more modular and reuseable by separating them out. For example:
`.nav-icon`
`.h1`